source/_docs/ecosystem/nginx.markdown

```diff
@@ -110,7 +109,7 @@ http {
 
 
         # These shouldn't need to be changed
-        listen 443 default_server;
+        listen [::]:443 default_server ipv6only=off; # if your nginx version is >= 1.9.5 you can also add the "http2" flag here
```

Comment:
I'm still quite certain this will make 443 a plain http port. Are you sure leaving out the `ssl` flag here is correct?